app/src/main/java/org/mozilla/focus/widget/BrowserNavigationOverlay.kt

```diff
@@ -117,16 +116,20 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
     override fun onClick(view: View?) {
         val event = NavigationEvent.fromViewClick(view?.id) ?: return
         var value: String? = null
+
+        val isTurboButtonChecked = turboButton.isChecked
```

Comment:
Do we need these values? Can you just use `isTurboEnabled` or `isPinButtonChecked` after it's set, without creating this additional value?

Reply:
You can do that but it's more error-prone – there are two steps to autocomplete that you can get wrong each time it's typed. Having it as one local variable means it's less likely to be mistyped. Also, (this doesn't matter since we're on the UI thread but if the code is refactored...) if the isChecked value is updated async, this ensures we're using the same value for both fields.

I'm going to leave it as is, but yeah, I'm conflicted about the best approach here.